src/bootstrap/native.rs

```diff
@@ -165,7 +165,7 @@ impl Step for Llvm {
 
         let llvm_exp_targets = match builder.config.llvm_experimental_targets {
             Some(ref s) => s,
-            None => "AVR",
+            None => "AVR;M68k",
```

Comment:
`config.toml.example` also also be updated with the new default value for `llvm.experimental-targets`.

Reply:
Fixed.